sklearn/cluster/spectral.py

```diff
@@ -218,6 +217,10 @@ def spectral_clustering(affinity, n_clusters=8, n_components=None,
         approach which is less sensitive to random initialization. See
         the 'Multiclass spectral clustering' paper referenced below for
         more details on the discretization approach.
+    copy : bool, optional, default: True
```

Comment:
new line before

Reply:
Okay, thanks for the comment. Please let me know if you have more suggestions for the code.